simple_rpc/simple_rpc.py

```diff
@@ -175,13 +175,17 @@ def close(self):
 
 
 class SocketInterface(Interface):
-    def __init__(self, device, baudrate=9600):
+    def __init__(self, device, baudrate=9600, autoconnect=True):
         """
         :arg str device: Device name.
         :arg int baudrate: Baud rate.
         """
         super().__init__(device, baudrate)
 
+        if autoconnect:
+            self.open()
+            self._close()
```

Comment:
There needed to be some way for `SocketInterface` to call `get_methods`. This is kind of similar to how `autoconnect` is working but since `_get_methods` is private it cannot be specified manually?